string grasp_entry
bool visualize_grasps
---
bool success
rail_grasping/GraspModel grasp
